integration_tests/test_math.py

```diff
@@ -72,13 +72,22 @@ def test_ldexp():
 
 
 def test_fabs():
-    i: f64
-    j: f64
+
+
+    # i: f64
+    # j: f64
+    # i = fabs(10.3)
+    # j = fabs(-10.3)
+    # assert abs(i - j) < eps
```

Comment:
Are you planning uncomment these tests?

Reply:
No, I was going to remove them in the next commit after your review. I'll do that now